docs/source/components/nodes/tof.rst

```diff
@@ -45,6 +45,24 @@ Inputs and Outputs
 - ``amplitude`` - :ref:`ImgFrame`
 - ``error`` - :ref:`ImgFrame`
 
+ToF Settings
+############
+
+In :ref:`ToF depth` example we allow users to quickly configure the following ToF settings:
+
+- FFPN Correction; It's a process that corrects the fixed pattern noise (FPN) of the ToF sensor. It's enabled by default for best performance.
+- Wiggle Correction: It's a process that corrects the wiggle effect of the ToF sensor. It's enabled by default for best performance.
+- Temperature Correction: It's a process that corrects the temperature effect of the ToF sensor. It's enabled by default for best performance.
+- Optical Correction: It's a process that corrects the optical effect (On -> ToF returns distance represented by Green Line), so it matches :ref:`StereoDepth` depth reporting.
+- Phase Unwrapping - Process that corrects the phase wrapping effect of the ToF sensor. You can set it to [0..4]. The higher the number, the longer the ToF range, but it also increases the noise.
```

Comment:
There is another error threshold config to filter out motion blur due to different sampling times of csptures.
Also median filter

Reply:
@SzabolcsGergely is that the `setMinAmplitude`? Could we describe these in depthai-core, so they are auto-included in docs as well?
![image](https://github.com/luxonis/depthai-python/assets/18037362/d5f03b2c-bdce-42f3-9499-342b654c8037)

Reply:
No, setminamplitude has been removed. Check tof decoding custom branch